### DISCUSSION TITLE
autotrain process suddenly interrupted

### DISCUSSION THREAD

**Discussion by fatihayUtopia** [huggingface/autotrain-advanced#744] [Q&A]
ı was training a model with autotrain and after one day i got an error and autotrain stopped "ERROR conda.cli.main_run:execute(125): `conda run /bin/bash -c pip uninstall -y autotrain-advanced && pip install -U autotrain-advanced && autotrain app --host 0.0.0.0 --port 7860 --workers 1` failed. (See above for error)" 
dataset name: fatihay/CombinedDataset
base model: Qwen/Qwen2-1.5B-Instruct.
Has the training process been reset? Can I continue where I left off?
I will pay for the training period that was suddenly interrupted

**Comment by abhishekkrthakur** [comment]
it seems like the training never started if you got this error. so, you can either factory rebuild the training space or try again? if it happens again, please let me know so we can investigate the issue.

**Reply by fatihayUtopia** [reply]
to try again i have to wait for it for days and pay money for t4

**Reply by abhishekkrthakur** [reply]
this error means the space was never started. so no gpu was used and nothing to pay. or am i misunderstanding something? 🤔

**Reply by fatihayUtopia** [reply]
ı was training the base model using nvidia t4 medium. Yesterday i started train and just now loss rate was 1.6 and train process was %43 and suddenly got that error and has been stopped the process

**Reply by abhishekkrthakur** [reply]
could you please share the link to space?

**Reply by fatihayUtopia** [reply]
fatihay/autotrain-advanced
